packages/storybook/src/stories/ino-icon-button/ino-icon-button.stories.ts

```diff
@@ -166,6 +185,73 @@ export const Variations = () => html`
   </div>
 `;
 
+export const CSSProperties = (args) => {
+  return html`
+    <style>
+      .customizable-icon-button {
+        --ino-icon-button-background-disabled-color: ${args.inoIconButtonBackgroundDisabledColor};
+        --ino-icon-button-icon-disabled-color: ${args.inoIconButtonIconDisabledColor};
+        --ino-icon-button-size: ${args.inoIconButtonSize};
+        --ino-icon-button-icon-size: ${args.inoIconButtonIconSize};
+        --ino-icon-button-icon-color: ${args.inoIconButtonIconColor};
+        --ino-icon-button-background-color: ${args.inoIconButtonBackgroundColor};
+        --ino-icon-button-icon-active-color: ${args.inoIconButtonIconActiveColor};
+        --ino-icon-button-background-active-color: ${args.inoIconButtonBackgroundActiveColor};
+      }
+    </style>
+    <ino-icon-button
+      class="customizable-icon-button"
+      disabled="${args.disabled}"
+      filled="${args.filled}"
+      color-scheme="primary"
+      activated="${args.activated}"
+      icon="time"
+    >
+    </ino-icon-button>
+  `;
+};
+CSSProperties.args = {
```

Comment:
@janivo this might be something we could also address in the template generator. Would be nice if we could: 
 * extract the name of the css vars from the config of the element
 * define a set of CssVarTypes (PxBased, Color) so that we only have to say which css has what type.

We don't need to address this here, but lets add some more background to https://github.com/inovex/elements/issues/453 then.